### PR TITLE
Add version tags

### PR DESCRIPTION
**Pull request by deronjohnson**
@jseldess, here's an implementation of version tags based on @kuanluo's screens. `<span class="version-tag">` is for in content tags and `<div class="version-tag">` is for tags in the table of contents, because the table of contents code seems to strip the text from spans. I added examples on the /dev/information-schema page, under user_privileges.

### CODE REVIEW

File: v1.1/information-schema.md [hunk 1]
```diff
@@ -148,9 +148,9 @@ Column | Description
 `TABLE_TYPE` | Type of the table: `BASE TABLE` for a normal table, `VIEW` for a view, or `SYSTEM VIEW` for a view created by CockroachDB.
 `VERSION` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table.
 
-### user_privileges
+### user_privileges <div class="version-tag">New in v1.1</div>
```
Comment (jseldess) [inline v1.1/information-schema.md]: @deronjohnson, it's nice that the icon shows up in the toc, but why doesn't it show up beside the header on the page as well?

Comment (deronjohnson) [inline v1.1/information-schema.md]: Sorry about that, I implied that you didn't want them to show up based off the screenshots in Asana. I can make it display there as well if that's what you'd like.

Comment (jseldess) [inline v1.1/information-schema.md]: Yes, that'd be great if you could make it display beside headers, too. Thanks, @deronjohnson.

File: v1.1/information-schema.md [hunk 2]
```diff
@@ -148,9 +148,9 @@ Column | Description
...
 
-The `user_privileges` view identifies global [privileges](privileges.html).
+<span class="version-tag">New in v1.1:</span> The `user_privileges` view identifies global [privileges](privileges.html).
```
Comment (jseldess) [inline v1.1/information-schema.md]: For this PR, let's remove this `<span>` and leave the `<div>` next to the heading. Once that's done, I'll merge. Thanks, @deronjohnson!

**Comment by jseldess** [comment]
LGTM. Thanks @deronjohnson.

**Pull request merged**